packages/admin/cms-admin/src/dam/Table/FolderTableRow.tsx

```diff
@@ -29,24 +29,28 @@ interface FolderTableRowProps extends DamConfig {
         hide: () => void;
     };
     archived?: boolean;
+    isNew?: boolean;
+    scrollIntoView?: boolean;
 }
 
 export interface DamDragObject {
     item: GQLDamFileTableFragment | GQLDamFolderTableFragment;
 }
 
-const StyledFolderTableRow = styled(TableBodyRow)<TableBodyRowProps & { $activeHoverStyle: boolean; $archived: boolean }>`
+const StyledFolderTableRow = styled(TableBodyRow)<TableBodyRowProps & { $activeHoverStyle: boolean; $archived: boolean; $highlightAsNew: boolean }>`
     height: 58px;
 
     outline: ${({ $activeHoverStyle, theme }) => ($activeHoverStyle ? `solid 1px ${theme.palette.primary.main};` : "none")};
     background: ${({ theme, $activeHoverStyle, $archived }) => {
         if ($activeHoverStyle) {
-            return "rgba(41,182,246,0.1)";
+            return alpha(theme.palette.primary.main, 0.1);
         } else if ($archived) {
             return theme.palette.grey[50];
         }
         return "none";
     }};
+    box-shadow: ${({ $highlightAsNew, theme }) => ($highlightAsNew ? `inset 0 0 99999px ${alpha(theme.palette.success.main, 0.4)}` : "none")};
```

Comment:
Why not use a background color? Because of the transition?

Reply:
Yes, because of the transition. 

If I would use a background, I would have to remove the transition after the highlighting. Then a timing issue occurs because the transition is removed before the fade out happened.

Reply:
@jamesricky is this the correct way to achieve such an animation?

Reply:
Here is an example of how it would work with a pseudo-element: https://codesandbox.io/s/multiple-background-animations-on-same-element-tj3164?file=/index.jsx 

Maybe this makes it more understandable in the code that the element's background is changing than when using `box-shadow`. 

But this is also a bit hacky and a lot more code, so I'm also fine with the `box-shadow` method.

Reply:
@jamesricky Changed it to a before pseudo element with background